charts/scalardl/files/grafana/scalardl_grafana_dashboard.json

```diff
@@ -0,0 +1,677 @@
+          "interval": "",
+          "legendFormat": "{{pod}} - {{quantile}}",
+          "refId": "B"
+        },
+        {
+          "exemplar": true,
+          "expr": "scalardl_stats_ledger_certificate_registration_request{quantile=\"0.95\",pod=~\"$pod\"}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "{{pod}} - {{quantile}}",
+          "refId": "C"
+        },
+        {
+          "exemplar": true,
+          "expr": "scalardl_stats_ledger_certificate_registration_request{quantile=\"0.98\",pod=~\"$pod\"}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "{{pod}} - {{quantile}}",
+          "refId": "D"
+        },
+        {
+          "exemplar": true,
+          "expr": "scalardl_stats_ledger_certificate_registration_request{quantile=\"0.99\",pod=~\"$pod\"}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "{{pod}} - {{quantile}}",
+          "refId": "E"
+        },
+        {
+          "exemplar": true,
+          "expr": "scalardl_stats_ledger_certificate_registration_request{quantile=\"0.999\",pod=~\"$pod\"}",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "{{pod}} - {{quantile}}",
+          "refId": "F"
+        }
+      ],
+      "title": "Certificate Registration Request",
+      "type": "timeseries"
+    }
+  ],
+  "refresh": false,
+  "schemaVersion": 30,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "current": {
+          "selected": false,
+          "text": "default",
+          "value": "default"
+        },
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": null,
+        "multi": false,
+        "name": "datasource",
+        "options": [],
+        "query": "prometheus",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "type": "datasource"
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "All",
+          "value": "$__all"
+        },
+        "datasource": "${datasource}",
+        "definition": "label_values(pod)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": null,
+        "multi": false,
+        "name": "pod",
+        "options": [],
+        "query": {
+          "query": "label_values(pod)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "/.*scalardl-ledger.*/",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      }
+    ]
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Scalar DL / Overview",
+  "uid": "scalardl-002",
+  "version": 23
+}
```

Comment:
Can you add the following?
Also, some of `request_failure` are still missing since failure didn't occur, but please add them as well.
```
scalardl_stats_ledger_contract_execution_request{quantile="0.5",} 0.546811857
scalardl_stats_ledger_contract_execution_request{quantile="0.75",} 0.546811857
scalardl_stats_ledger_contract_execution_request{quantile="0.95",} 0.546811857
scalardl_stats_ledger_contract_execution_request{quantile="0.98",} 0.546811857
scalardl_stats_ledger_contract_execution_request{quantile="0.99",} 0.546811857
scalardl_stats_ledger_contract_execution_request{quantile="0.999",} 0.546811857
scalardl_stats_ledger_contract_execution_request_count 257.0
scalardl_stats_ledger_contract_execution_request_success 257.0
scalardl_stats_ledger_execution_abort_request{quantile="0.5",} 0.0
scalardl_stats_ledger_execution_abort_request{quantile="0.75",} 0.0
scalardl_stats_ledger_execution_abort_request{quantile="0.95",} 0.0
scalardl_stats_ledger_execution_abort_request{quantile="0.98",} 0.0
scalardl_stats_ledger_execution_abort_request{quantile="0.99",} 0.0
scalardl_stats_ledger_execution_abort_request{quantile="0.999",} 0.0
scalardl_stats_ledger_execution_abort_request_count 72.0
scalardl_stats_ledger_execution_abort_request_success 72.0
scalardl_stats_ledger_asset_proof_retrieval_request{quantile="0.5",} 0.0
scalardl_stats_ledger_asset_proof_retrieval_request{quantile="0.75",} 0.0
scalardl_stats_ledger_asset_proof_retrieval_request{quantile="0.95",} 0.0
scalardl_stats_ledger_asset_proof_retrieval_request{quantile="0.98",} 0.0
scalardl_stats_ledger_asset_proof_retrieval_request{quantile="0.99",} 0.0
scalardl_stats_ledger_asset_proof_retrieval_request{quantile="0.999",} 0.0
scalardl_stats_ledger_asset_proof_retrieval_request_count 72.0
scalardl_stats_ledger_asset_proof_retrieval_request_success 1.0
scalardl_stats_ledger_asset_proof_retrieval_request_failure 71.0
scalardl_stats_ledger_ledger_validation_request{quantile="0.5",} 0.064910166
scalardl_stats_ledger_ledger_validation_request{quantile="0.75",} 0.064910166
scalardl_stats_ledger_ledger_validation_request{quantile="0.95",} 0.064910166
scalardl_stats_ledger_ledger_validation_request{quantile="0.98",} 0.064910166
scalardl_stats_ledger_ledger_validation_request{quantile="0.99",} 0.064910166
scalardl_stats_ledger_ledger_validation_request{quantile="0.999",} 0.064910166
scalardl_stats_ledger_ledger_validation_request_count 1.0
scalardl_stats_ledger_ledger_validation_request_success 1.0

```

Reply:
The above issues has been corrected.
Please check it.
https://github.com/scalar-labs/helm-charts/pull/25/commits/748a778a1cd3225b62bf5509c790088a224d9b05